### ISSUE TITLE
Japanese vs. English label names

### ISSUE THREAD

**Issue by Rangi42** [pret/pokegold-spaceworld#77]
Currently the labels and constants are like this:

- Pokémon are entirely in Japanese
- Items are translated literally, or using their canon English translations when valid
- Moves are translated literally, or using their canon English translations when valid
- Maps are in Japanese with English descriptors (e.g. "BaadonHouse1", "ShizukanaOka")

So far PR #75 is planning to use more English names for maps, e.g. "BirdonHouse1", "SilentHillDungeon". (The Romaji town/city names will match up with TCRF.)

The question is how to handle Pokémon names. Should the original 1-151 switch to using English names? Should any Gen 2 mons use their final English names, and if so, how similar do they have to be to the final design? Yadoking is obviously Slowking, but Chikorita's Japanese name changed from SW97 to the final (as did its middle evolution), and Two-Head/Ariados changed its name and design.

(I'd lean towards keeping all-Japanese names—picking the Gen 2 ones case by case would be too subjective, and just using English for Gen 1 would lead to e.g. Slowpoke evolving into Yadoking—but any discussion of label/constant names can go here.)

**Comment by SatoMew** [comment]
Something else of note is how Game Freak apparently misspelled some of the Pokémon names even in Japanese like クルス *Cruz* (Portuguese for "cross"). Given that its evolutions are アクア *Aqua* and アクエリア *Aquaria*, Cruz doesn't make sense and they likely meant to type クルーズ *Cruise*.

**Comment by Rangi42** [comment]
IMO it would make sense to correct the obvious typos, like Waste → West or Cruz → Cruise.

**Comment by SatoMew** [comment]
I don't know where Waste came from, but it'll be fixed for sure! 😊

**Comment by ketsuban** [comment]
> Should the original 1-151 switch to using English names?

Yes. Most people don't have the Japanese names memorised. (I sure don't - when the subject came up on Discord I tried to recall one off the top of my head and I got it wrong. (It was Spearow, but I remembered *Onisuzuku* rather than *Onisuzume*.)

> Should any Gen 2 mons use their final English names, and if so, how similar do they have to be to the final design? Yadoking is obviously Slowking, but Chikorita's Japanese name changed from SW97 to the final (as did its middle evolution), and Two-Head/Ariados changed its name and design.

Personally: a Pokemon should get its English names if it's recognisable as what we got in the final release. (As such, the list would go Chikorita, *Hanamogura*, Meganium, *Honooguma*, *Dynabear*, *Volbear*, *Kurusu*, *Aqua*, *Aquaria*, Hoothoot, Noctowl, Mareep, Flaaffy, Ampharos, *Mikon*, *Monja*, *Jaranra*, Mantine, Qwilfish, *Shibirefugu*, Pichu, Cleffa, Igglybuff, Quagsire, Natu, Xatu, *Gyopin*, Marill, *Manbo1*, *Ikari*, *Gurotesu*, Crobat, *Para*, Spinarak, Ariados, Skarmory, *Animon*, *Hinaazu*, Sunflora, Phanpy, Donphan, *Twins*, Girafarig, Smeargle, *Koonya*, *Rinrin*, *Berurun*, Politoed, Slowking, Unown, Ledyba, Ledian, *Puchikorn*, Espeon, Umbreon, *Turban*, *Betobaby*, Remoraid, Octillery, Tyrogue, Hitmontop, *Pudi*, Hoppip, Skiploom, Jumpluff, *Baririina*, Smoochum, Elekid, Magby, Bellossom, *Tsubomitt*, Miltank, *Bomushikaa*, Delibird, *Kotora*, *Raitora*, *Madame*, *Norowara*, *Kyonpan*, Murkrow, Blissey, Scizor, *Plucks*, Houndour, Houndoom, *Wolfman*, *Warwolf*, Porygon2, *Nameeru*, Steelix, Kingdra, Raikou, Entei, Suicune, Sneasel, Ho-oh, Togepi, Snubbull, Aipom, Leafeon.)

> Something else of note is how Game Freak apparently misspelled some of the Pokémon names even in Japanese

That's speculation, and I find it implausible that Game Freak would misspell a name they made up in their own language. That said, romanising it as "Cruz" or "Cruise" is also presumptuous - the only honest representation until we can interrogate a Game Freak employee or something and find out more is *Kurusu*.

**Comment by iimarckus** [comment]
> Should the original 1-151 switch to using English names?

I think this is without question.

> Should any Gen 2 mons use their final English names, and if so, how similar do they have to be to the final design?

@ketsuban’s proposal seems good enough. I would also settle for only translating names that are completely unchanged from the final. But I think this is a good list.

**Comment by jd-0** [comment]
I have [this version](https://github.com/jd-0/pokegold-spaceworld/blob/979f2601cf21547cd05001e1afc878eb4e5ecc27/data/pokemon/dex_entries.asm) here of the DexEntries with the 151 English names if needed!

**Comment by Caverns4** [comment]
I don't see why we  shouldn't use the English names where available. It improves consistency with pokegold/pokecrystal, and it also makes things more comprehensible to a reader. Example: Yadon is Slowpoke's Japanese name, but an english-speaker might not always know that. This disassemblies comments/documentation are in English, after all.

**Comment by BreavyTF2** [comment]
English names should be used if the name used matches the final JP name for the Pokémon.
